skaled/main.cpp

```diff
@@ -1046,8 +1050,9 @@ int main( int argc, char** argv ) try {
 
     setupLogging( loggingOptions );
 
-    const size_t nCpuCount = skutils::tools::cpu_count();
-    size_t nDispatchThreads = nCpuCount * 2;
+    // we do not really use these threads anymore
```

Comment:
where were these threads used? should we remove them completely if we don't use them?

Reply:
They were used to HTTP JSON-RPC in the past, before we moved to proxygen server

They are still used in websockets.  We do not know of any customers that use websockets, so fewer number of threads is OK.

We need to move to proxygen for websockets.  When we do it, we can remove these completely.